loader.js

```diff
@@ -18,7 +18,13 @@ nconf.argv().env().file({
 });
 
 var	pidFilePath = path.join(__dirname, 'pidfile');
-var outputLogFilePath = path.join(__dirname, 'logs/output.log');
+
+var outputLogFilePath = path.join(__dirname, nconf.get('logFile') || 'logs/output.log');
+
+var logDir = path.dirname(outputLogFilePath);
+if (!fs.existsSync(logDir))
+	fs.mkdirSync(path.dirname(outputLogFilePath), {recursive: true});
```

Comment:
can you wrap this in {} to please linter

Reply:
Run `npm run lint` to check that everything is fixed when you're done. Thanks!